src/ConnectedMode/Suppressions/IServerIssuesStore.cs

```diff
@@ -48,14 +48,14 @@ public interface IServerIssuesStore
     internal interface IServerIssuesStoreWriter : IServerIssuesStore
     {
         /// <summary>
-        /// Updates the IsResolved status of the issue with the specified key 
+        /// Updates the IsResolved status of the issues with the specified keys
         /// </summary>
-        /// <remarks>If the issue key cannot be matched to an issue in the store it will be ignored.
+        /// <remarks>If an issue key cannot be matched to an issue in the store it will be ignored.
         /// Design decision: if we can't find an issue we could have decided to go to the server and
```

Comment:
Minor: I think the design decision comment is irrelevant now that we do do it in the updater